format/src/main/scala/scala/meta/internal/fmt/SyntacticGroup.scala

```diff
@@ -15,7 +14,7 @@ object SyntacticGroup {
     case object ParamTyp extends Type { def precedence = 0 }
     case object Typ extends Type { def precedence = 1 }
     case object AnyInfixTyp extends Type { def precedence = 1.5 }
-    case class InfixTyp(op: String) extends Type { def precedence = 2 }
+    case class InfixTyp(operator: String) extends Type { def precedence = 2 }
     case object RefineTyp extends Type { def precedence = 3 }
     case object WithTyp extends Type { def precedence = 3.5 }
```

Comment:
Should we remove WithTyp if it's not used?

Reply:
https://github.com/olafurpg/scala-syntax/pull/15#discussion_r171026847